ee/clickhouse/util.py

```diff
@@ -25,6 +27,13 @@ def _assertNumQueries(self, func):
     def assertNumQueries(self, num, func=None, *args, using=DEFAULT_DB_ALIAS, **kwargs):
         return self._assertNumQueries(func)
 
+    # :NOTE: Update snapshots by passing --snapshot-update to bin/tests
```

Comment:
Q: Does this updates snapshots for all tests with `assertQueryMatchesSnapshot` ? or do I need to specifically mention tests so it only creates/updates snapshots for them?

Reply:
It will update all.